docs/zoo.json

```
@@ -6,6 +6,12 @@
     "street": "Calle de la piruleta",
     "number": "123",
     "postal_code": "00000"
-  },
-  "animals": null
-}
+  }
+
+  ,"animals":[
```

The formatting seems a little bit off.

What about something like this?
```suggestion
  },
  "animals":[
```